src/ctia/http/routes/bulk.clj

```diff
@@ -57,27 +68,47 @@
                    :sighting       read-sighting
                    :ttp            read-ttp) %))
 
+(defn with-long-id-fn
+  "return the with-long-id function provided an entity type key"
+  [k]
+  (case k
```

Comment:
I was just following the pattern of the surrounding code.  Perhaps open an issue and submit a PR to refactor this NS?